### PR TITLE
Adding bio and avatar Sofia Papastamkou, closes #864

### LINKED ISSUE #864: Onboarding FR editor checklist - Sofia
This is a individual onboarding editor checklist for @spapastamkou, initially found [here 
](https://github.com/programminghistorian/jekyll/wiki/Onboarding-Checklist-for-New-Editors). Please do also have a look at the [specific FR team milestone ](https://github.com/programminghistorian/jekyll/milestone/7) and the [French Translation infrastructure pages ](https://github.com/programminghistorian/jekyll/issues/836).
## What we will do

Within 2 weeks of your joining, our current team will ensure that you have been granted access to the following services used by the Project Team:

- [x] Our Google Group email so you can communicate with other editors (@acrymble)
- [x] Our Google Analytics account for monitoring web traffic (@acrymble)
- [x] Our Twitter team (@acrymble)
- [x] Our Github Repository so you can upload materials to the site (@mdlincoln)
- [x] Our Skype group so that you can participate in our monthly calls (@walshbr)
- [x] We will also write a blog post welcoming you publicly to the team.
- [x] partner you with an existing editor to 'shadow' the editing process so you can get a feel for it.

## What you should do

There are a few tasks you will need to perform to make sure you know how to use our system, and also to make sure your details appear on the Project Team page. All of these tasks are encouraged, but some may be optional:

### communication

- [x] set up a GitHub account and inform @mdlincoln of your username (mandatory)
- [x] set up a twitter account (optional)
- [x] set up a Skype account and inform @walshbr (mandatory)
- [x] inform @acrymble of your Google-activated email (mandatory)
- [x] write an email to the other editors using the Google group (say hello!) (mandatory)

### technical

We rely heavily on Github for communicating and conducting our editorial processes. You will need to get familiar with it and the way we use it. We recognise that this can take time, so if you would like one-on-one training, please let someone know and this will be arranged. To make sure you can do all required tasks, try the following activities on our Github /jekyll website.

- [x] contribute to one open ticket on our github repository (mandatory)
- [x] create and assign to yourself a new ticket on the github repository to ensure you know how to do this. Add a relevant label. (mandatory)
- [x] make a technical contribution on your own. This will normally be adding your bio to the /data/ph_authors.yml file, and uploading a square black and white avatar (.png) of yourself to /avatars). You will then submit a pull request for approval so that this bio information can be added to the Project Team page (mandatory)
- [x] Read all sections of the /contribute pages of the website, including author, reviewer, and editor guidelines
- [x] Bookmark the /jekyll (https://github.com/programminghistorian/jekyll) and /submissions (https://github.com/programminghistorian/ph-submissions) github sites so you can find them easily
- [x] Read the 'Making Technical Contributions' wiki page: https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions

## The Culture of the Project

We have invited you to join the team because we value your opinions. We encourage all of our editors to contribute to discussions on github or via email, and to civilly express their views, even if they disagree with any or all other members. This right to express one's opinions is central to our project. Please take this to heart.

### PR DESCRIPTION
**Pull request by spapastamkou**
I added my bio to the /data/ph_authors.yml file, and uploaded a square black and white avatar of myself to /avatars). As one of the technical tasks described in #864

### CODE REVIEW

**Comment by acrymble** [comment]
Thanks for doing this @spapastamkou.

Since your bio needs to also appear on the Spanish project team page, you will have to request a translation from someone on our Spanish team (unless you can speak Spanish). You can do that in your pull request by alerting the Spanish team and highlighting that this pull request cannot be accepted until we have a translation. This is as easy as saying @mariajoafana @vgayolrs @arojascastro @jamotilla @jenniferisasi would one of you please translate this?

**Comment by arojascastro** [comment]
Ok, but shoudn't we have a French version first of all?

**Comment by spapastamkou** [comment]
@acrymble thank you. For the Spanish translation I thought it was enough to pick one of the Spanish-speaking members as reviewer but I'll be more explicite in the future. 
@arojascastro Sure but should we not better wait that a FR edition actually exists before? Or perhaps translate all bios in French could be part of the preparatory work for the FR edition.

**Comment by acrymble** [comment]
@spapastamkou you will have to do it eventually, so you could translate the snippet into French now if you would like to save time later.

File: _data/ph_authors.yml [hunk 1]
```diff
@@ -971,3 +971,18 @@
...
+  twitter: s_papastamkou
+  github: spapastamkou
+  team: true
+  team_start: 2018
+  institution: CNRS
+  sortname: Papastamkou
+  affiliation: Maison européenne des sciences de l'homme et de la société Lille Nord de France
+  bio:
+        en: | 
+        Sofia Papastamkou is a historian and Research Engineer in Digital Humanities at the French National Center for Scientific Research (CNRS).
```
Comment (mdlincoln) [inline _data/ph_authors.yml]: A small change is required: you must indent all lines after `en: |` so that they line up with the `|`

**Review by mdlincoln — CHANGES_REQUESTED** [review]

**Review by arojascastro — APPROVED** [review]
spanish translation added. correction by Matthew pending

**Review by amsichani — APPROVED** [review]

**Comment by mdlincoln** [comment]
Thanks @spapastamkou - now that you've done that, you'll see that "all checks have passed" on this PR which means the site builds successfully.

**Review by mdlincoln — APPROVED** [review]

**Comment by spapastamkou** [comment]
Thank you!

>

**Comment by drjwbaker** [comment]
Welcome aboard @spapastamkou! (I always think once someone is on the about page then it is properly official that they are part of the team)

**Comment by spapastamkou** [comment]
thank you @drjwbaker and all, it is great to be part of THIS team.

**Pull request merged**